### PR TITLE
Add azure as platform and hypershift as subplatform

### PR DESCRIPTION
**Pull request by morenod**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Adding azure and hypershift cli platform/subplatform

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: hcp-burner.ini [hunk 1]
```diff
@@ -9,10 +9,13 @@ delay_between_batch = 60
 batch_size = 1
 watcher_delay = 60
 
-enable_workload = False
-workload = cluster-density-ms
-workload_repo = "https://github.com/cloud-bulldozer/e2e-benchmarking.git"
-workload_script_path = "workloads/kube-burner-ocp-wrapper"
+enable_workload = True
+workload_script = run.sh
+workload = cluster-density-v2
```
[the remark this answers is not in the document]
Comment (morenod) [reply hcp-burner.ini:14]: restored

File: hcp-burner.py [hunk 2]
```diff
@@ -85,7 +85,7 @@
                     raise
 
     if str(platform.environment["cleanup_clusters"]).lower() == "true":
-        platform = utils.get_cluster_info(platform)
+        #        platform = utils.get_cluster_info(platform)
```
[the remark this answers is not in the document]
Comment (morenod) [reply hcp-burner.py:88]: restored

File: libs/utils.py [hunk 3]
```diff
@@ -132,16 +132,14 @@ def get_cluster_info(self, platform):
             platform.environment["clusters"][cluster_name]["path"] = platform.environment["path"] + "/" + cluster_name
             platform.environment["clusters"][cluster_name]["kubeconfig"] = platform.environment["clusters"][cluster_name]["path"] + "/kubeconfig"
             platform.environment['clusters'][cluster_name]['workers'] = int(platform.environment["workers"].split(",")[(loop_counter - 1) % len(platform.environment["workers"].split(","))])
-        cluster_mc = platform.get_mc(platform.get_cluster_id(cluster_name))
-        platform.environment["mc_kubeconfig"] = platform.environment["path"] + "/kubeconfig_" + cluster_mc
```
[the remark this answers is not in the document]
Comment (mukrishn) [reply libs/utils.py:136]: PR #29  PTAL

**Review by mukrishn — APPROVED** [review]
lgtm, tested.

**Pull request merged**